docs/automated-workflows/dagman-advance-functionality.rst

```diff
@@ -64,16 +67,17 @@ double quotes. To use a double quote mark within a *string*, escape
 the double quote mark with the backslash character (``\"``). To add
 the backslash character itself, use two backslashes (``\\``).
 
-Multiple ``key=value`` pairs can be specified in a single **VARS**
-line with a space in between each pair. Multiple individual **VARS**
+Multiple ``key=value`` pairs can be specified in a single :dag-cmd:`VARS`
+line with a space in between each pair. Multiple individual :dag-cmd:`VARS`
 lines can also be used for the same node.
```

Comment:
Not to be full English teacher here, but I'd strongly prefer active voice throughout, e.g.

"A single VARs line may have multiple space-separated ``key=value`` pairs.